### PR TITLE
Add License and Readme

### PR DESCRIPTION
**Pull request by avlaskin**
I added basic License and Readme, if you would like to change them later on - please do so, but for now we need them even in a basic form.

### CODE REVIEW

File: Dive Computer/main.c [hunk 1]
```diff
@@ -34,41 +77,50 @@ static void TIMER_0_wakeup_task_cb(const struct timer_task *const timer_task)
 	ticks++;
 }
 
-void TIMER_0_example(void)
+void TIMER_0_Init(void)
```
Comment (Arrowana) [inline Dive Computer/main.c]: Init or init?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (Arrowana) [inline LICENSE:1]: Pierre Arrowana -> Pierre Duval

File: Dive Computer/driver_init.c [hunk 3]
```diff
@@ -123,18 +136,32 @@ void AON_SLEEP_TIMER0_register_isr(void)
 	*temp = (uint32_t)AON_SLEEP_TIMER0_Handler;
 }
 
-void system_init(void)
+uint8_t getSoundPin() 
+{
+	return GPIO(GPIO_PORTA,7);
+}
+
+void system_init(bool spiEnabled, bool i2cEnabled, bool soundEnabled)
```
Comment (Arrowana) [inline Dive Computer/driver_init.c:144]: We have define and variable for SPI_ENABLED, I2C_ENABLED...

Comment (avlaskin) [reply Dive Computer/driver_init.c:144]: Let's dedicate a file for configuration and pin settings.

File: Dive Computer/main.c (hunk 1 above)
Comment (avlaskin) [reply Dive Computer/main.c]: Fixed. Let's add simple code convention that we are following Atmel autogenerate code style.

File: LICENSE (hunk 2 above)
Comment (avlaskin) [reply LICENSE:1]: Fixed.

**Pull request merged**